src/content/blog/metriq_release.md

```diff
@@ -0,0 +1,83 @@
+    <a href='https://riacs.usra.edu/quantum/nisqc-nl'><img src="https://res.cloudinary.com/dcz4ywuer/image/upload/v1690841822/ivtg9hqbkbvhrlwic9pp.png" alt='USRA partner logo' style='display: inline-block; width: 140px; height: auto; max-width: 180px; max-height: 180px; padding: 12px;' /></a><br />
+    <a href='https://cambridgequantum.com/'><img src="https://res.cloudinary.com/dcz4ywuer/image/upload/v1690841822/lp80k7qb6pamjrpimhu6.png" alt='Cambridge Quantum Computing partner logo' style='display: inline-block; width: 140px; height: auto; max-width: 180px; max-height: 180px; padding: 12px;' /></a>
+    <a href='https://www.riken.jp/en/'><img src="https://res.cloudinary.com/dcz4ywuer/image/upload/v1690841822/w5ahq5qov7dbvdqh2akx.png" alt='Riken University logo' style='display: inline-block; width: 140px; height: auto; max-width: 180px; max-height: 180px; padding: 12px;' /></a>
+    <a href='https://strangeworks.com/'><img src="https://res.cloudinary.com/dcz4ywuer/image/upload/v1690841822/nese9jpfylu1hffmomju.png" alt='Strangeworks logo' style='display: inline-block; width: 140px; height: auto; max-width: 180px; max-height: 180px; padding: 12px;' /></a>
+    <a href='https://sqms.fnal.gov/'><img src="https://res.cloudinary.com/dcz4ywuer/image/upload/v1690841822/tlx1aeltoctpdgs061b3.png" alt='SQMS logo' style='display: inline-block; width: 140px; height: auto; max-width: 180px; max-height: 180px; padding: 12px;' /></a><br />
+    <a href='https://www.quantonation.com/'><img src="https://res.cloudinary.com/dcz4ywuer/image/upload/v1690841822/dyybccql1d15pxbo0ecu.png" alt='Quantonation logo' style='display: inline-block; width: 140px; height: auto; max-width: 180px; max-height: 180px; padding: 12px;' /></a>
+    <a href='https://agnostiq.ai/'><img src="https://res.cloudinary.com/dcz4ywuer/image/upload/v1690841822/bz21od6ut84g5qkrefg4.png" alt='Agnostiq logo' style='display: inline-block; width: 140px; height: auto; max-width: 180px; max-height: 180px; padding: 12px;' /></a>
+    <a href='https://super.tech/'><img src="https://res.cloudinary.com/dcz4ywuer/image/upload/v1690841822/zjkllt4hgs7ayus7nwio.png" alt='Super.Tech logo' style='display: inline-block; width: 140px; height: auto; max-width: 180px; max-height: 180px; padding: 12px;' /></a>
+</div>
+
+Whether you are a professional quantum hardware researcher, a quantum software developer, or a hobbyist eager to learn more about progress in the field, Metriq is free and open to community contributions. Come see how the latest quantum technologies measure up!
+
+If you’d like to learn more or have help making a submission then reach out to us at metriq@unitary.fund.
+
+<div style="text-align: center;font-weight: bold;">
+    metriq@unitary.fund
+</div>
+<br />
```

Comment:
This seems redundant to have the metriq email twice here.

Reply:
It was the original post content, but you have a point.